AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -1677,6 +1705,8 @@ protected SharedPreferences restorePreferences() {
             mGestureTapTop = Integer.parseInt(preferences.getString("gestureTapTop", "12"));
             mGestureTapBottom = Integer.parseInt(preferences.getString("gestureTapBottom", "2"));
             mGestureLongclick = Integer.parseInt(preferences.getString("gestureLongclick", "11"));
+            mGestureVolumeUp = Integer.parseInt(preferences.getString("gestureVolumeUp", "11"));
```

Comment:
We should refactor this to take a `ViewerCommandDef` (not necessarily in this PR, but the meaning of the strings isn't clear)
```suggestion
            mGestureVolumeUp = Integer.parseInt(preferences.getString("gestureVolumeUp", "0"));
```